Vagrantfile.erb

```diff
@@ -0,0 +1,133 @@
+<% config[:vagrantfiles].each do |vagrantfile| %>
```

Comment:
@therobot You might have a global kitchen file in your `~/.kitchen/config.yml` that might be referencing it.